neurom/features/tests/test_neuronfunc.py

```diff
@@ -176,6 +176,9 @@ def test_sholl_crossings_simple():
     nt.eq_([2, 4, 5],
            list(_nf.sholl_crossings(SIMPLE, center, radii=radii)))
 
+    nt.eq_([2, 2, 2],
```

Comment:
@asanin-epfl I added the test here in fact! I'm surprising myself that I added a test!

Reply:
Then I'm very surprised that it passed with the function from my comment above https://github.com/BlueBrain/NeuroM/pull/871#discussion_r599594919. Anyway. Check this new function, and if it works, lets use it.

Reply:
yes, weird. I recall it was not working when I tried... something may have changed. Let's go like that then.